Makefile

```diff
@@ -32,6 +32,7 @@ BUNDLE_IMG ?= controller-bundle:$(VERSION)
 IMG_NAME ?= quay.io/csiaddons/volumereplication-operator
 IMG_TAG ?= latest
 IMG=${IMG_NAME}:${IMG_TAG}
+image_arch_list=linux/amd64,linux/ppc64le,linux/s390x
```

Comment:
make this a variable like IMG_NAME not everyone is interested in building multi-arch images locally for testing.

Reply:
I added an option to choose if to build multi-arch or not, so if someone will build multi-arch images he will use it, and if someone will choose the one with only x86 won't use it, so I think it can be good that way after the change

Reply:
the only problem is that now with multi-arch the image building takes more than expected. just want to avoid that

Reply:
Discard this one.